quickstart.md

```diff
@@ -100,30 +100,23 @@ There are two layers of security available before the charger has the opportunit
 
 Once a charger has a Charging Station entry and its password has been set, you can connect it to the security profile 1 websocket server at `ws://localhost:8082`.
 
-If you want to add a password for security profile 1, send the following request to the CitrineOS API.
+If you want to add a password for security profile 1 and 2, send the following request to the CitrineOS API.
 
 ```shell
-curl --location --request PUT 'localhost:8080/data/monitoring/variableAttribute?stationId=cp001&setOnCharger=true' \
+curl --location --request POST 'localhost:8080/data/configuration/password?callbackUrl=csms.pro/api/notifications' \
 --header 'Content-Type: application/json' \
 --data '{
-    "component": {
-        "name": "SecurityCtrlr"
-    },
-    "variable": {
-        "name": "BasicAuthPassword"
-    },
-    "variableAttribute": [
-        {
-            "value": "testing-citrine"
-        }
-    ],
-    "variableCharacteristics": {
-        "dataType": "passwordString",
-        "supportsMonitoring": false
-    }
+  "stationId": "cp001",
+  "password": "DbkTu048=xueA69@-dB94hjV*fGdyQOzgVuhhH1L",
```

Comment:
Can you make clear that this is a "clear text" password and not hashed yet. Maybe make it some nice readable string

Reply:
@ChrisWeissmann done, hope it's clearer now!